### PR TITLE
Added alias for datacentres method and fixed typos

### PR DESCRIPTION
**Pull request by brejoc**
In some places datacentre was used instead of datacenter.

### CODE REVIEW

**Comment by ghost** [comment]
Seems like my Britishness got the better of me. 

Unfortunately, with this package now being released and ready for use in a production environment, I can't release a breaking change like this without increasing the major version of the library. I propose we have two options:

1) Leave it as it is.
2) Add another method to the `HetznerCloudClient` class called `datacenters` (which returns the result from the `datacentres` function). In layman's terms, create an alias. 

Whatever route we take, thank you for your contribution.

**Comment by brejoc** [comment]
Okay, I see! :laughing: Then I'll change the PR, so that we have an alias. Would you prefer to have `datacenters` or `datacentres` as an alias.

**Comment by ghost** [comment]
I'd prefer datacenters to be the alias - cheers!

**Comment by brejoc** [comment]
Sorry, something went wrong with the merge. I have to take care of this tomorrow.

**Comment by brejoc** [comment]
@elsyms Okay, now we should be good to go. :smile:

**Comment by brejoc** [comment]
@elsyms Is there anything else that needs to be done here?

**Comment by ghost** [comment]
Thanks for your work on this. Will run the tests myself and deploy a new version sometime today.
